ItemList/Source/ItemList.Api.Services/Bootstrapper.cs

```diff
@@ -10,9 +10,10 @@ public class Bootstrapper : IBootstrapper
     {
         public void RegisterTypes(IDependencyInjectionContainer container)
         {
-            container.RegisterRequestScoped<IItemUrlHelper, ItemUrlHelper>();
-            container.RegisterRequestScoped(() => (HttpRequestMessage)HttpContext.Current.Items["MS_HttpRequestMessage"]);
-            container.RegisterRequestScoped<IDatabaseConfiguration, DatabaseConfiguration>();
+            container
```

Comment:
This is really great. You can make these [expression bodied function](https://imgflip.com/i/1nlubh), can't you?